tests/email_framework/test_templates.py

```
@@ -0,0 +1,381 @@
+                instance.validate_data(invalid_data)
+    
+    def test_practice_report_theme_selection(self):
+        """Test that theme is selected based on location."""
+        locations_themes = [
+            ("Baytown", "baytown"),
+            ("Humble", "humble"),
+            ("BAYTOWN", "baytown"),
+            ("humble", "humble"),
+        ]
+        
+        for location, expected_theme in locations_themes:
```

Ruff (0.12.2)

Loop control variable `location` not used within loop body

Rename unused `location` to `_location`

(B007)